docs/manifest.json

```diff
@@ -163,6 +163,13 @@
 						}
 					]
 				},
+				{
+					"title": "Coder Desktop",
+					"description": "Use Coder Desktop to access your workspace like it's a local machine",
+					"path": "./user-guides/desktop/index.md",
+					"icon_path": "./images/icons/computer-code.svg",
+					"state": ["beta"]
```

Comment:
leaving this as `beta` for now because we don't have an `earlyaccess` badge

Reply:
FYI @bpmct.
@EdwardAngert I would opt for `alpha` over `beta` in this case. We do support that.